config/odp-linux-generic.conf

```diff
@@ -33,6 +33,10 @@ shm: {
 	# When using process mode threads, this value should be set to 0
 	# because the current implementation won't work properly otherwise.
 	num_cached_hp = 0
+
+	# Allocate internal shared memory using a single virtual address space.
+	# Set to 1 to enable using process mode.
+	single_va = 0
```

Comment:
This seems to be duplicate of PR #706. Just temporary?

Given the scope of these changes, we really need to test with `single_va = 1` as well as the default 0 here.

Reply:
>This seems to be duplicate of PR #706. Just temporary?

Yes, the first three commits are temporarily copied from PR #706 as this patch set builds on top of that PR.